### PR TITLE
Add operator Spatial and Logical

### PR DESCRIPTION
**Pull request by Vquillien**
Last pull reqest for the class FesToXml.

### CODE REVIEW

File: filter/fes-2.0.2/src/test/java/org/orbisgis/orbisdata/filter/fes_2_0_2/TestFes2_0_2.java [hunk 1]
```diff
@@ -101,10 +100,10 @@ public void testXmlToSqlSortBy() throws JAXBException {
         xml = TestFes2_0_2.class.getResourceAsStream("filter_Sorting.xml");
         Object elementObject = "filter_Sorting";
 
-        XmlToSql(elementObject).toString();
+        XmlToSql(elementObject);
 
         //Error : objectFromFilterXml is null
-        XmlToSql(null).toString();
+        XmlToSql(null);
     }
```
Comment (SPalominos) [inline filter/fes-2.0.2/src/test/java/org/orbisgis/orbisdata/filter/fes_2_0_2/TestFes2_0_2.java]: An assert is probably missing

File: filter/fes-2.0.2/src/test/java/org/orbisgis/orbisdata/filter/fes_2_0_2/TestFes2_0_2.java [hunk 2]
```diff
@@ -173,8 +172,7 @@ public void testXmlToSqlFilterComparison() throws JAXBException {
         xml = TestFes2_0_2.class.getResourceAsStream("filter_PropertyIsLikeWithoutExpression.xml");
         element = (JAXBElement) unmarshaller.unmarshal(xml);
 
-        Assert.assertEquals(XmlToSql(element).toString(), "");
-
+        XmlToSql(element);
     }
```
Comment (SPalominos) [inline filter/fes-2.0.2/src/test/java/org/orbisgis/orbisdata/filter/fes_2_0_2/TestFes2_0_2.java]: An assert is probably missing

File: filter/fes-2.0.2/src/test/java/org/orbisgis/orbisdata/filter/fes_2_0_2/TestFes2_0_2.java [hunk 3]
```diff
@@ -191,5 +189,105 @@ public void testXmlToSalFilterFunction() throws JAXBException {
...
+        element = (JAXBElement) unmarshaller.unmarshal(xml);
+
+        Assert.assertEquals(XmlToSql(element).toString(), "!( ST_Disjoint( element , 100 , 200 )");
+
+        //error : the object is not an instance of JaxBElement
+        xml = TestFes2_0_2.class.getResourceAsStream("filter_UnrecognizedObject.xml");
+        element = (JAXBElement) unmarshaller.unmarshal(xml);
+
+        XmlToSql(element);
+
```
Comment (SPalominos) [inline filter/fes-2.0.2/src/test/java/org/orbisgis/orbisdata/filter/fes_2_0_2/TestFes2_0_2.java]: An assert is probably missing

File: filter/fes-2.0.2/src/test/java/org/orbisgis/orbisdata/filter/fes_2_0_2/TestFes2_0_2.java (hunk 1 above)
Comment (Vquillien) [inline filter/fes-2.0.2/src/test/java/org/orbisgis/orbisdata/filter/fes_2_0_2/TestFes2_0_2.java]: For those three cases, I need to test if the message of the logger is launch. It's why I don't use an assert. But I don't know if there is an other way.

**Pull request merged**